## Rules for Primer Design

**Date:** 04-13-2021

**Author:** Nishimura lab

--- 

### Rules for all primers:

- Ideal annealing length of 18-22 base pairs
- Ideal melting temperature of 55°C for annealing region
- GC content should be 40-60%
- 2-3 G’s or C’s should be in the last five bases of the 3’ end. (No more than 3)
- Secondary structure should be avoided, ie hairpins, self-dimers, and cross dimers.
- No extensively repeated dinucleotides or trinucleotides, for example “ATATAT” or ATGATG”
- No runs longer than four bases, for example AGGGGGCT.

### Tips for all primers:

- Use the IDT oligo calculator to check melting temperature and look for secondary structures.
- BLAST everything. BLAST your primers against all organisms and particularly do a comparative BLAST of the plasmids or genome you are amplifying from.
- Use software such as APE, SnapGene, or an online tool to help design primers and look at their parameters.

### Primer Dilution

Be careful not to cross contaminate primers. This is a very difficult problem to troubleshoot.

<ins>100 uM stock solution
- Write the number of the primer in ethanol resistant marker on the cap.
- Find the real number of nanomoles on the side of the tube and write this number on the IDT packing list.
- Multiply the number of nanomoles by 10 and add this volume in microliters of sterile ddH<sub>2</sub>O to the tube.
- Vortex tube for ~10-30 seconds.

<ins>10 uM working stock
- Write the primer number in ethanol resistant marker on the lid of an Eppendorf tube.
- Add 90ul of sterile ddH<sub>2</sub>O to the Eppendorf tube.
- Add 10ul of vortexed 100 uM stock primer solution to the tube.
- Vortex for ~10-30 seconds.

Both dilutions of the primers can then be placed in the freezer. There are separate boxes for 100uM and 10uM stocks.
