Sanger Sequencing

Date: \_\_\_\_\_\_\_\_\_\_\_\_\_\_

Which primers will be used for sequencing?

What will be sequenced?

- Add 2.5 uL of each 10 uM sequencing primer to a PCR tube.
- Add the DNA to be sequenced (See below for quantities based on DNA type and size)
- Add diH<sub>2</sub>O to 15 uL.
- Log on to genewiz.com using your username and password.
- Click “Create sequencing order”
- Click “Same day,” “online form,” “pre-mixed,” enter the number of samples, and click “create new form.”
- Enter the name of your DNA, select whether it is a plasmid of PCR product, enter the length in nucleotides of the entire plasmid/PCR product, enter your primer name, and select save and next. Don’t worry about the other columns.
- Review what you have entered and select “next step.”
- The next page should be auto filled. Click “Save PO for next time use” box and then select “next step”
- Review the submission info and submit the order.
- Print off two copies of the receipt.
- Place one copy of the receipt in your notebook.
- Label each of the PCR tubes with the sequencing reagents in them as EN01, EN02, EN03, etc in the order that you entered them into the online form. This should be written in ethanol proof marker down the side of the tube.
- Tape the tubes face-down to the second receipt.
- Fold the receipt over the tubes, place them in a ziplock bag, and put them in the Genewiz sequencing submission box.
- Sequences should come back for analysis around 3PM the next day.

Sanger Sequencing Outline

In Sanger sequencing, or chain-terminator sequencing, a primer of your design that is complementary to a region just 5’ of the region of DNA being sequenced binds to your DNA sample. This then allows a DNA polymerase to bind to the sample and begin elongating the DNA strand. Normally when this occurs using normal deoxynucleotides the DNA has a 3’ hydroxyl group that is required for catalysis and can be used during the next round of polymerization to elongate the DNA strand. However, in Sanger sequencing a combination of deoxynucleotides and dideoxynucleotides are present. When the polymerase catalyzes the addition of a dideoxynucleotide to the 3’ end of a DNA the polymerization is terminated, because there is no longer a 3’ hydroxyl group on the end of the DNA strand to promote further elongation. By adding fluorescent dyes to the dideoxynucleotides it is then possible to visually label each DNA strand only at the point where polymerization was terminated by the dideoxynucleotide. These strands can then be separated by length using capillary gel electrophoresis. Since the shortest strands of DNA will run through the gel the fastest, we can then get a fluorescent readout of which deoxynucleotide is at each position in the DNA sequence based on when they pass the fluorescence detector. Using this method, we can usually get a sequencing read of ~600-1000nts (with an average of ~750). Due to the limitations in the technique there are a few things to keep in mind: (1) The first ~20-30 nucleotides read by the machine will be nonsense as the leftover reagents flow through the capillary, (2) homopolymeric, or long stretches of the same nucleotide will often be predicted to be incorrect due to difficulty in computational analysis of which base is at which location (also known as base calling), because the chromatographic peak will appear very blobby. These can often be better interpreted manually, and (3) sequencing quality decreases significantly as the length of the sequencing reaction is increased due to a decreased signal to noise ratio as more and more of the template DNA is dideoxy terminated.


